pkg/mimir/modules.go

```diff
@@ -1024,6 +1031,54 @@ func (t *Mimir) initUsageStats() (services.Service, error) {
 	return t.UsageStatsReporter, nil
 }
 
+func (t *Mimir) initContinuousTest() (services.Service, error) {
+	start := func(ctx context.Context) error {
+		logger := util_log.Logger
+		var err error
+
+		level.Info(logger).Log("msg", "initing continuous-test")
+		// Setting the environment variable JAEGER_AGENT_HOST enables tracing.
+		if trace, err := tracing.NewFromEnv("mimir-continuous-test", jaegercfg.MaxTagValueLength(16e3)); err != nil {
+			level.Error(logger).Log("msg", "Failed to setup tracing", "err", err.Error())
+		} else {
+			defer trace.Close()
+		}
+		level.Info(logger).Log("msg", "continuous-test setup tracing happened")
+
+		// Run the instrumentation server.
+		registry := prometheus.NewRegistry()
+		registry.MustRegister(version.NewCollector("mimir_continuous_test"))
+		registry.MustRegister(collectors.NewGoCollector())
+		level.Info(logger).Log("msg", "prom registry created for ct")
+
+		i := instrumentation.NewMetricsServer(t.Cfg.ContinuousTest.ServerMetricsPort, registry)
+
+		level.Info(logger).Log("msg", "metrics server created for ct")
+		if err := i.Start(); err != nil {
+			level.Error(logger).Log("msg", "Unable to start instrumentation server", "err", err.Error())
+			util_log.Flush()
+		}
```

Comment:
This entire block can be removed. We shouldn't start a dedicated instrumentation server for continuous test. We should just use Mimir one, which is already started at this point.